src/components/expanded-state/ChartExpandedState.js

```diff
@@ -82,7 +83,9 @@ function useJumpingForm(isLong) {
   }, [isLong, setOptions, jumpToShort, jumpToLong]);
 }
 
-export const ChartExpandedStateSheetHeight = chartExpandedAvailable
+export const ChartExpandedStateSheetHeight = android
+  ? deviceUtils.dimensions.height - 40
```

Comment:
Isn't this gonna cause a big white gap if no chart on android?

Reply:
hm, I'll do it a bit different way